pkg/subctl/cmd/gather/cni.go

```diff
@@ -97,8 +97,6 @@ func CNIResources(info Info, networkPlugin string) {
 			switch networkPluginCNIType[networkPlugin] {
 			case typeIPTables:
 				logIPTablesCmds(info, pod)
-			case typeOvn:
```

Comment:
let's keep the "typeOvn: " with no action, otherwise we will get the typeUnknown "failure".